common/mqtt.py

```diff
@@ -0,0 +1,109 @@
+import json
+import paho.mqtt.publish as publish
+import threading
+
+
+class MqttSender:
+    def __init__(self, config: dict):
+        self.batch = []
+        self.counter = 0
```

Comment:
Is this field redundant? `self.counter` is just `len(self.batch)`

Reply:
[bd512b1](https://github.com/magley/my-pi-home/pull/6/commits/bd512b14ca2255faab587a060310bcdd08fc0f9d)